test/elastic/integration/index_test.exs

```diff
@@ -0,0 +1,27 @@
+defmodule Elastic.Integration.IndexTest do
+  use Elastic.IntegrationTestCase
+  alias Elastic.{HTTP, Index}
+
+  @tag integration: true
+  test "close/1 & open/1" do
+    Answer.index(1, %{text: "Hello world!"})
+
+    cat_result = get_index_response(HTTP.get("_cat/indices?format=json"), Index.name("answer"))
```

Comment:
Oh never mind, I just tried it out myself! Sorry, should've done that before asking.